src/components/select/select.ts

```diff
@@ -365,6 +364,17 @@ export class Select extends BaseInput<string[]> implements AfterViewInit, OnDest
     });
   }
 
+  /**
+   * Close the select interface.
+   */
+  close() {
```

Comment:
I think name should be `dismiss` as for convention. @manucorporat is this right?